src/AppInstallerCommonCore/Public/winget/MsixManifest.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+
+#include <AppxPackaging.h>
+#include <AppInstallerStrings.h>
+#include <AppInstallerVersions.h>
+#include <wrl/client.h>
+
+#include <string>
+
+namespace AppInstaller::Msix
+{
+    using string_t = Utility::NormalizedString;
```

Comment:
nit: Putting these in the root of the namespace could lead to collisions if someone else `using`s this in.  Mostly a problem for something generic like `string_t`.